### PR TITLE
chore: code split tenant API CE

### PR DESCRIPTION
**Pull request by berzerkeer**
## Description

We shouldn't expose tenant config on CE , so on CE,  we should only return the necessary user permissions hard coded on the saga.

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Manual

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by mohanarpit — APPROVED** [review]

File: app/client/src/ce/sagas/tenantSagas.tsx [hunk 1]
```diff
@@ -1,25 +1,19 @@
-import TenantApi from "@appsmith/api/TenantApi";
 import {
-  ReduxActionErrorTypes,
   ReduxActionTypes,
-} from "@appsmith/constants/ReduxActionConstants";
-import { ApiResponse } from "api/ApiResponses";
-import { call, put } from "redux-saga/effects";
-import { validateResponse } from "sagas/ErrorSagas";
+  ReduxActionErrorTypes,
+} from "ce/constants/ReduxActionConstants";
```
Comment (ankitakinger) [inline app/client/src/ce/sagas/tenantSagas.tsx:4]: this will be `@appsmith/constants/ReduxActionConstants`

Comment (berzerkeer) [reply app/client/src/ce/sagas/tenantSagas.tsx]: done

**Review by ankitakinger — APPROVED** [review]
LGTM

**Pull request merged**